### ISSUE TITLE
The input box does not work in arm64 version!

### ISSUE THREAD

**Issue by FreddyFeuerstein** [AOF-Dev/MCinaBox#555]
**Describe the bug**
The inputbox (top left ingame) does not open the virtual keyboard on press.

**To Reproduce**
1. Go ingame
2. Click on 'message box'

**Expected behavior**
It should open the virtual keyboard.

**Smartphone (please complete the following information):**
 - Device: Samsung galaxy a40
 - OS: Android 10 (with root and custom kernel)
 - App Version 0.1.4-p4

**Comment by AyHa1810** [comment]
Well, it works for me when I use hackers keyboard.

**Comment by LegacyGamerHD** [comment]
> Well, it works for me when I use hackers keyboard.

@AyHa1810 i tetesed gboard and smanung keyboard both dont work it isnt clickable

**Comment by AyHa1810** [comment]
@LegacyGamerHD yeah, those apps doesn't works for me as well. I use `Hacker's Keyboard` which works for me. 

BTW, your engrishu is very nice.

**Comment by TheGameYIYI** [comment]
@AyHa1810  probe ese teclado, pero aun asi no me funciona al hacer click al cuadro

**Comment by TheGameYIYI** [comment]
@AyHa1810 I tried that keyboard, but it still doesn't work for me when I click the box

**Comment by TheGameYIYI** [comment]
@AyHa1810    I tried that keyboard, but it still doesn't work for me when I click the box

**Comment by AyHa1810** [comment]
@TheGameYIYI this a type of bug which acts in some devices, not in others.

**Comment by alex9900123** [comment]
> Well, it works for me when I use hackers keyboard.

How do I use the hacker keyboard or where do I install it?

**Comment by WzzyXP** [comment]
Use pc keyboard feature and enable type

**Comment by kaiwalyar** [comment]
#735
